prep-html-syntax/index.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+<body></body>
+  <h1 style="color:blue">My favorite Start Wars character is Ashoka!</h1>
+  <img src="https://www.denofgeek.com/wp-content/uploads/2020/05/star-wars-ahsoka-tano.png?fit=1280%2C720" alt="Ashoka, Star Wars">
+  <p style="color:blueviolet">"She is a great Jedi who never thinks of herself when someone needs her assistance!"</p>
```

Comment:
Hi Daniel, be sure to structure your HTML skeleton with a `<head></head>` at the top and a `<body></body>` to contain all of the visible content.

https://lfz-lms.herokuapp.com/code-exercises/Learning-Fuze/curriculum/prep-html-skeleton